migtests/scripts/functions.sh

```diff
@@ -451,22 +451,27 @@ import_data_to_source_replica() {
 }
 
 import_data_file() {
-	yb-voyager import data file --export-dir ${EXPORT_DIR} \
-		--target-db-host ${TARGET_DB_HOST} \
-		--target-db-port ${TARGET_DB_PORT} \
-		--target-db-user ${TARGET_DB_USER} \
-		--target-db-password ${TARGET_DB_PASSWORD:-''} \
-		--target-db-schema ${TARGET_DB_SCHEMA:-''} \
-		--target-db-name ${TARGET_DB_NAME} \
-		--disable-pb true \
-		--send-diagnostics=false \
-		--parallel-jobs 3 \
-		$* || {
-			cat ${EXPORT_DIR}/metainfo/dataFileDescriptor.json
-			exit 1
-		}
+    args="
+    --export-dir ${EXPORT_DIR}
+    --target-db-host ${TARGET_DB_HOST}
+    --target-db-port ${TARGET_DB_PORT}
+    --target-db-user ${TARGET_DB_USER}
+    --target-db-password ${TARGET_DB_PASSWORD:-''}
+    --target-db-schema ${TARGET_DB_SCHEMA:-''}
+    --target-db-name ${TARGET_DB_NAME}
+    --disable-pb true
+    --send-diagnostics=false
+    "
+
+    # Check if RUN_USING_ADAPTIVE_PARALLELISM is true
+    if [ "${RUN_USING_ADAPTIVE_PARALLELISM}" = "true" ]; then
```

Comment:
Should we have this for `import-data` as well, so that we can test both the commands as and when require.